plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -1213,6 +1226,104 @@ public function backfill( array $args = array(), array $assoc_args = array() ) {
 		);
 	}
 
+	/**
+	 * Show the list of WooCommerce-aware plugins known to be compatible, incompatible or without compatibility declaration for HPOS. Note that inactive plugins will always be listed in the "uncertain" list.
+	 *
+	 * [--include-inactive]
+	 * : Include inactive plugins in the list.
+	 *
+	 * [--display-filenames]
+	 * : Print plugin file names instead of plugin names.
+	 *
+	 * @since 9.1.0
+	 *
+	 * @param array $args       Positional arguments passed to the command.
+	 * @param array $assoc_args Associative arguments (options) passed to the command.
+	 */
+	public function compatibility_info( array $args = array(), array $assoc_args = array() ) {
```

Comment:
(Doesn't require action, this is more a high-level thought) I wonder if it would make most sense to support a family of commands for feature compatibility, rather than potentially registering commands for individual features like HPOS?

That is, rather than having an HPOS specific command:

```sh
wp wc hpos compatibility-info
```

Have a compatibility command that can work with any feature:

```sh
wp wc compatibility <feature-slug>
```

Reply:
This is certainly a good idea but I think it's better to implement this in a separate PR.